Jenkinsfile

```diff
@@ -8,7 +8,7 @@
 
 pipeline {
     agent {
-        node { label 'Sphinx' }
+        dockerfile { label 'DockerCI' }
```

Comment:
Looks good but you can specify the label for nodes supporting Docker in the global jenkins settings instead.  _Manage Jenkins_ -> _Configure System_ -> _Docker Label_

Reply:
Updated.